apps/prelaunch/models/base.py

```diff
@@ -169,7 +169,7 @@ def grant_early_access(self):
             'Early access has been enabled for {} <{}>'.format(
                 self.full_name, self.email
             ),
-            level='info',
+            level='debug',
```

Comment:
This changes it for every message. We only want to change the priority to lower if it is the dev environment.

One uses https://github.com/hacktoolkit/django-htk/blob/6c2d7c8a63253bf0e1f9398b72b42ba9dc8581f4/apps/prelaunch/models/base.py#L142

another uses https://github.com/hacktoolkit/django-htk/blob/6c2d7c8a63253bf0e1f9398b72b42ba9dc8581f4/apps/prelaunch/models/base.py#L168

We should probably just use one of them, and not both.

Something in this file is not working correctly: https://github.com/hacktoolkit/django-htk/blob/master/utils/notifications.py